acceptance/analyzer_test.go

```diff
@@ -27,36 +27,28 @@ import (
 	h "github.com/buildpacks/lifecycle/testhelpers"
 )
 
-const (
-	readWriteImage = "image-readable-writable"
-	onlyReadImage  = "image-readable"
-	onlyWriteImage = "image-writable"
-	noAccessImage  = "noAccessImage"
-)
-
 var (
 	analyzerBinaryDir    = filepath.Join("testdata", "analyzer", "analyze-image", "container", "cnb", "lifecycle")
 	analyzeDockerContext = filepath.Join("testdata", "analyzer", "analyze-image")
 	analyzeImage         = "lifecycle/acceptance/analyzer"
 	analyzerPath         = "/cnb/lifecycle/analyzer"
 	cacheFixtureDir      = filepath.Join("testdata", "analyzer", "cache-dir")
 	daemonOS, daemonArch string
-	readOnlyRegistry     *ih.DockerRegistry
-	authRegistry         *ih.DockerRegistry
-	customRegistry       *ih.DockerRegistry
+	testRegistry         *ih.DockerRegistry
 	registryNetwork      string
 	fixtures             analyzeFixtures
 )
 
 type analyzeFixtures struct {
-	authRegAppImage       string
-	authRegAuthConfig     string
-	authRegCacheImage     string
-	authRegOtherAppImage  string
-	daemonAppImage        string
-	daemonCacheImage      string
-	readOnlyRegAppImage   string
-	readOnlyRegCacheImage string
+	readWriteAppImage      string
+	regAuthConfig          string
+	readWriteCacheImage    string
+	readWriteOtherAppImage string
+	daemonAppImage         string
+	daemonCacheImage       string
+	readOnlyAppImage       string
+	readOnlyCacheImage     string
+	inaccessibleImage      string
```

Comment:
```suggestion
	daemonAppImage         string
	daemonCacheImage       string
	inaccessibleImage      string
	readOnlyAppImage       string
	readOnlyCacheImage     string
	readWriteAppImage      string
	readWriteCacheImage    string
	readWriteOtherAppImage string
	regAuthConfig          string
```
(nit: sorted)

Reply:
Fixed in the following [commit](https://github.com/buildpacks/lifecycle/pull/685/commits/7ce8bb8297e15af0aab609e3df4c462172972ef9)